samples/mcpclient/tsup.config.js

```diff
@@ -0,0 +1,13 @@
+/** @type {import('tsup').Options} */
+module.exports = {
+  dts: true,
+  minify: false,
+  bundle: false,
+  sourcemap: true,
+  treeshake: true,
+  splitting: true,
+  clean: true,
```

Comment:
since this is a node sample that we know won't be used in the browser, I typically don't use tsup. Wondering what your thoughts are on that or if there are some benefits of it I don't know about outside of bundling